### ISSUE TITLE
az network lb frontend-ip create does not work in parallel

### ISSUE THREAD

**Issue by jaslawinMs** [Azure/azure-cli#13736]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Environment summary**

**Additional context**

**Comment by jaslawinMs** [comment]
```
$ipsInJson | % -Parallel {
    $ipId = $_.publicIp.id
    $name = 'fe-' + ($ipId -split '/')[-1]
    az network lb frontend-ip create --subscription $using:subscription -g $using:rg --lb-name $using:lbName -n $name --public-ip-address $ipId
} 
```
if executed for at least 2 IPs, one configuration is created, and quickly it is replaced by another one, so the result is only the last finishing command 'wins'
It only works if the loop is throttled to 1, which is sequential execution.

az version: 
{
  "azure-cli": "2.1.0",
  "azure-cli-command-modules-nspkg": "2.0.3",
  "azure-cli-core": "2.1.0",
  "azure-cli-nspkg": "3.0.4",
  "azure-cli-telemetry": "1.0.4",
  "extensions": {}
}

**Comment by yonzhan** [comment]
add to S171

**Comment by MyronFanQiu** [comment]
@jaslawinMs Hi. Based on my knowledge, we don't support parallel command right now. I would mark it as feature request. Add @jiasli @Juliehzl for awareness.

**Comment by jaslawinMs** [comment]
Cool, maybe we should write it in the doc explicitly?

Is this somehow related https://github.com/Microsoft/azure-pipelines-tasks/issues/8561 ?

Also, I dig a little bit now and see this: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-cli?view=azure-devops
With useGlobalConfig

Maybe I used it incorrectly and should switch it off (not global config)?

Best
js

**Comment by MyronFanQiu** [comment]
@jaslawinMs  I double check your issue. Based on my knowledge, one lb can only have one frontend-ip. Are you talking about the overlap between different frontend-ips for the same lb?

**Comment by jaslawinMs** [comment]
No. It can have many. As I wrote I can add many if I do it sequentially

Sent from Outlook Mobile<https://aka.ms/blhgte>

________________________________

**Comment by MyronFanQiu** [comment]
@jaslawinMs I see. We cannot support such scenario right now. Thanks for raising this issue. We would consider the design to see how we can support this feature.
